packages/protocol/test-sol/integration/CompileValidatorMock.t.sol

```diff
@@ -8,7 +8,7 @@ import "forge-std/console.sol";
 import "@test-sol/unit/governance/validators/mocks/ValidatorsMock.sol";
 
 contract CompileValidatorMock is Test {
```

Comment:
Is this test suite useful for anything, could it be completely removed?

Reply:
This is necessary for forge to compile
import "@test-sol/unit/governance/validators/mocks/ValidatorsMock.sol";